olm-catalog/serverless-operator/1.6.0/serverless-operator.v1.6.0.clusterserviceversion.yaml

```diff
@@ -91,35 +91,96 @@ spec:
+      blue-green testing with gradual traffic rollout following best practices.
+    - **Use any programming language or runtime of choice**: From Java, Python, Go
+      and JavaScript to Quarkus, SpringBoot or Node.js.
+    - **Automatic scaling:** No need to configure number of replicas, or idling.
+      Scale to zero when not in use, auto scale to thousands during peak, with
+      built-in reliability and fault-tolerance.  
+    - **Event Driven Applications**: Build loosely coupled & distributed
+      applications connecting with a variety of built-in or third-party event
+      sources connectors powered by Operators.  
+    - **Ready for the hybrid cloud**: True portable serverless running anywhere
+      OpenShift runs, that is on premise or on any cloud. Leverage data locality
+      and SaaS when needed.
 
-    Knative Serving builds on Kubernetes to support deploying and
-    serving of serverless applications and functions. Serving is easy
-    to get started with and scales to support advanced scenarios. The
-    Knative Serving project provides middleware primitives that
-    enable:
+    ## Components & APIs
+
+    This operator provides the following components:
+
+    ### Knative Serving
+
+    Knative Serving builds on Kubernetes to support deploying and serving of
+    applications and functions as serverless containers. Serving is easy to get
+    started with and scales to support advanced scenarios. Other features
+    includes: 
 
     - Rapid deployment of serverless containers
     - Automatic scaling up and down to zero
     - Routing and network programming
     - Point-in-time snapshots of deployed code and configurations
 
-    ## Prerequisites
+    ![](https://i.imgur.com/vqL48B8.png)
 
-    The Serverless Operator's provided APIs such as Knative Serving
-    have certain requirements with regards to the size of the underlying
-    cluster. See [Getting started with OpenShift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/serverless_applications/index#serverless-getting-started)
-    for more info.
+
+    ### Knative Client - `kn`
+
+    The Knative client `kn` is your door to the Knative world. It allows you to
+    create Knative resources interactively from the command line or from within
+    Shell scripts.
+
+    `kn` offers you:
+
+    - Full support for managing all features of Knative Serving (services,
+      revisions, traffic splits) 
+    - Growing support Knative eventing, closely following its development
+      (managing of sources & triggers) 
+    - A plugin architecture similar to that of kubectl plugins 
+    - A thin client-specific API in golang which helps in tasks like synchronously
+      waiting on Knative service write operations.  
+    - An easy integration of Knative into Tekton Pipelines by using kn in a Tekton
+      Task.
+
+    <!-- ### Knative Eventing
```

Comment:
I'm not sure if the HTML comments work in here. I'd leave this out just in case.